billiard/process.py

```diff
@@ -270,7 +263,7 @@ def _bootstrap(self):
                 except (OSError, ValueError):
                     pass
             old_process = _current_process
-            _set_current_process(self)
+            _current_process = self
```

Comment:
We need to set multirprocessing.current_process here, or logging `%(processName)` and other tools will not have the correct details.